workspaces.tf

```diff
@@ -24,10 +24,14 @@ locals {
     # Inherits defaults.
     test = {
       /* Settings specific to the test fleet/workspace. */
+      cf_zone_id = "14660d10344c9898521c4ba49789f563"
```

Comment:
Do you intend to switch `waku.test` to `status.im` domain separately?

Reply:
Removed